protected/widgets/menu/assets/menu.css

```diff
@@ -1,15 +1,6 @@
 /**
  * This CSS file is menu widget
  */
-.menu h1 {
```

Comment:
This block is removed here and added again later.

Reply:
It was duplicated, I've regrouped the two